tcpserver/TCPServer.h

```diff
@@ -84,6 +90,9 @@ class CTCPServerInt : public CTCPServerIntBase {
 	boost::asio::ip::tcp::acceptor acceptor_;
 
 	CTCPClient_ptr new_connection_;
+
+	bool logFirstTime(const std::string &ip_string);
```

Comment:
All functions start with a Uppercase letter (camel system)
Name is not very clear, could you rename it to (for instance) IsUserHereFirstTime(...)